src/plural.js

```diff
@@ -41,23 +44,40 @@ Globalize.prototype.plural = function( value ) {
  * default/instance locale.
  */
 Globalize.pluralGenerator =
-Globalize.prototype.pluralGenerator = function() {
-	var cldr, plural;
+Globalize.prototype.pluralGenerator = function(options) {
+	var cldr, plural, isOrdinal;
 
+	validateParameterTypePlainObject( options, "options" );
+
+	options = options || {};
 	cldr = this.cldr;
 
+	validateParameterTypePluralType( options.type, "options.type" );
+
 	validateDefaultLocale( cldr );
 
+	isOrdinal = (options.type === "ordinal");
+
 	cldr.on( "get", validateCldr );
-	cldr.supplemental( "plurals-type-cardinal/{language}" );
+	if (isOrdinal) {
```

Comment:
Please, replace `isOrdinal` with `options.type === "ordinal"`, which is pretty straightforward already.